src/lib/ngsi10/SubscribeContextRequest.cpp

```diff
@@ -57,7 +57,8 @@ std::string SubscribeContextRequest::check(const std::string& predetectedError,
       ((res = duration.check())                                                         != "OK") ||
       ((res = restriction.check(restrictions))                                          != "OK") ||
       ((res = notifyConditionVector.check(SubscribeContext, predetectedError, counter)) != "OK") ||
-      ((res = throttling.check())                                                       != "OK"))
+      ((res = throttling.check())                                                       != "OK") ||
+      ((res = maxFailsLimit.check())                                                    != "OK"))
```

Comment:
NGSIv1 is deprecated, so no need of modifying this file in anyway. The new functionality will be only for NGSIv2.

Please revert the change in this file.